.github/workflows/ci.yml

```diff
@@ -113,41 +136,46 @@ jobs:
         run: "cp development/creds.example.env development/creds.env"
       - name: "Linting: pylint"
         run: "poetry run invoke pylint"
+      - name: "Checking: App Config"
+        run: "poetry run invoke validate-app-config"
+      - name: "Checking: migrations"
+        run: "poetry run invoke check-migrations"
   unittest:
     needs:
-      - "pylint"
+      - "check-in-docker"
     strategy:
       fail-fast: true
       matrix:
-        python-version: ["3.7", "3.8", "3.9", "3.10"]
+        python-version: ["3.8", "3.11"]
+        db-backend: ["postgresql"]
```

Comment:
This is not going to be needed, along with anything else that is postres or mysql.

Reply:
Removed